### ISSUE TITLE
Mesentery torsion around the ileocolic vessels

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10115]
**Preferred term label:**
Mesentery torsion around the ileocolic vessels

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C1403952

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cecal volvulus

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000632
- label: Mesenteric torsion
- parents: [Abnormal mesentery morphology (HP:0100016)](https://hpo.jax.org/app/browse/term/HP:0100016)
- pmid: 
- def: Mesenteric torsion is a pathological rotation of the intestinal tract around the axis of the mesenteric root, resulting in rapid occlusion of the cranial mesenteric artery.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10115
- synonyms:
